### ISSUE TITLE
bug(respawning persons): captured person escorts revert back to their original owner

### ISSUE THREAD

**Issue by petervdmeer** [endless-sky/endless-sky#10379]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When a player encounters a Person-ship for a second time, after capturing an escort ship from such Person the first time (without destroying the flagship). Then the captured escort automatically gets transferred over to the Person, while still keeping an escort label.

### Steps to Reproduce

1. Increase the spawn-rate for person ships through `data/gamerules.txt`
2. Start a fight with a person with flagship and escorts
3. Capture an escort from the person (without destroying the flagship)
4. Land on a planet
5. Take off again, and search for the same person again
6. Notice your escort suddenly changing government, and jumping in with the person

### Expected Behavior

When I capture a ship, then I expect it to remain mine.

### Screenshots

N/A

### Link to save file

Saving the game, and then reloading it makes the issue disappear (because the saving action causes the ship to be cloned upon reloading)

### Operating System

Not relevant, should be reproducible on all operating systems

### Game Source

Built from source

### Game Version

ec9350673c729021329bcb556e83ca41c731e7c9

### Additional Information

The commit mentioned in this ticket is a from a recent, not-yet-merged PR, but the issue also exists on today's master.
Except that the author-ships on master cannot be captured, so that masks the issue.

**Comment by tibetiroka** [comment]
Part of me thinks this is hilarious, the other part is horrified by the number of issues we have with person ships.

**Comment by VaelophisNyx** [comment]
tbh I feel like person ships _should_ be captureable. It does make you hostile with the author faction anyway so, its not like its unpunished if you do it/try to do it

**Comment by Saugia** [comment]
The reason they aren't is because most of them are terribly overpowered and unbalanced. As rewards/representations of big contributors in-game, we wouldn't want to impose hard restrictions to the builds in-universe, and because they're too strong, we wouldn't make those ones capturable.

**Comment by VaelophisNyx** [comment]
> The reason they aren't is because most of them are terribly overpowered and unbalanced. As rewards/representations of big contributors in-game, we wouldn't want to impose hard restrictions to the builds/we wouldn't change the ships against these creators' wishes, and because they're too strong, we wouldn't make those ones capturable.

could raise the question then, of if they should *be* overpowered in the first place :P

**Comment by ThrawnCA** [comment]
> could raise the question then, of if they should _be_ overpowered in the first place :P

Yes, because it's more entertaining. They're not really supposed to participate in gameplay, they're supposed to roar into the system with a trail of light behind them and get your attention. 

Capturing them is... Well, not something a novice pilot could do, but I suppose it's also entertaining.
